models/wrf/tutorial/README.rst

```diff
@@ -233,13 +250,16 @@ bash  ``export BASE_DIR=<path_to_your_working_directory>``
    ::
 
        cd $BASE_DIR
-       wget http://www.image.ucar.edu/wrfdart/tutorial/wrf_dart_tutorial_23May2018_v3.tar.gz
-       tar -xzvf wrf_dart_tutorial_23May2018_v3.tar.gz
+       wget http://www.image.ucar.edu/wrfdart/tutorial/wrf_dart_tutorial_29Apr2024.tar.gz
+       tar -xzvf wrf_dart_tutorial_29Apr2024.tar.gz
 
    After untarring the file you should see the following directories:
    *icbc, output, perts,* and *template.* The directory names (case
    sensitive) are important, as the scripts rely on these local paths
-   and file names.
+   and file names. Please note that the perturbation, surface and initial 
+   condition files were derived from an earlier version (pre-4.0) of WRF/WPS/WRFDA
+   but still maintains compatibility with the (post-4.0, post-11.4.0) 
+   WRF-DART versions recommended to run this WRF assimilation example.  
```

Comment:
Step 6. ~line 845 has T rather than THM

https://github.com/NCAR/DART/blob/f8325c86412f43b049f8b47f6b611f74d005bf0c/models/wrf/tutorial/README.rst?plain=1#L945-L949

you could change it to this:

``MU, PH, PSFC, QRAIN, QCLOUD, QGRAUP, QICE, QNICE, QSNOW, QVAPOR, THM`` and ``T2``. 
 The example figure below shows the increments for temperature only. You can